lib/actions/configure.coffee

```diff
@@ -75,51 +76,53 @@ module.exports =
 			return if not isMounted
 			umount.umountAsync(params.target)
 		.then ->
-			reconfix.readConfiguration(CONFIGURATION_SCHEMA, params.target).then (data) ->
+			denymount params.target, (cb) ->
```

Comment:
It would have been nice to be able to use a disposer here to ensure the cleanup, but I couldn't find a sensible way to make it work :(